ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -53,6 +53,28 @@ tests:
   - mount_path: /usr/local/sd-cicd-aws-prod
     name: sd-cicd-aws-prod
   skip_if_only_changed: ^(?:docs|\.github)|\.md$|^(?:\.gitignore|.golang-ci.yml|OWNERS|LICENSE)$
+- as: rosa-stage-mcvw
+  run_if_changed: pkg/e2e/webhooks
+  commands: |
+    export GOFLAGS=""
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="hypershift"
```

Comment:
@jbpratt Is the plan to try to use `hypershift` cluster (if it supports webhooks) or do we want to stick with ROSA sts stage environment to test with?

Reply:
the plan was to _try_ to use hypershift because it would be quick but after the investigation this morning, I'll need to switch this to sts.